app/frontend/src/views/form/Manage.vue

```diff
@@ -1,274 +1,45 @@
+import { mapActions } from 'vuex';
 
-import { FormPermissions, NotificationTypes } from '@/utils/constants';
-import FormSettings from '@/components/designer/FormSettings.vue';
-import ShareForm from '@/components/forms/ShareForm.vue';
+import ManageForm from '@/components/forms/manage/ManageForm.vue';
+import ManageFormActions from '@/components/forms/manage/ManageFormActions.vue';
 
 export default {
   name: 'FormManage',
-  components: { ShareForm, FormSettings },
+  components: { ManageForm, ManageFormActions },
   props: {
     f: {
       type: String,
       required: true,
     },
   },
-  data() {
-    return {
-      deleteDialog: false,
-      formSettingsDisabled: true,
-      showDeleteDialog: false,
-      settingsFormValid: false,
-      settingsPanel: 0,
-      versionsPanel: 0,
-    };
-  },
-  computed: {
-    ...mapGetters('form', ['form', 'permissions']),
-
-    // Permission checks
-    canCreateDesign() {
-      return this.permissions.includes(FormPermissions.DESIGN_CREATE);
-    },
-    canDeleteForm() {
-      return this.permissions.includes(FormPermissions.FORM_DELETE);
-    },
-    canEditForm() {
-      return this.permissions.includes(FormPermissions.FORM_UPDATE);
-    },
-    canManageTeam() {
-      return this.permissions.includes(FormPermissions.TEAM_UPDATE);
-    },
-
-    currentVersion() {
-      return this.form.versions ? this.form.versions[0] : {};
-    },
-  },
+  computed: {},
   methods: {
     ...mapActions('form', [
-      'deleteCurrentForm',
       'getFormPermissionsForUser',
+      'fetchDrafts',
       'fetchForm',
-      'updateForm',
     ]),
-    ...mapActions('notifications', ['addNotification']),
-    cancelSettingsEdit() {
-      this.formSettingsDisabled = true;
-      this.fetchForm(this.f);
-    },
-    deleteForm() {
-      this.showDeleteDialog = false;
-      this.deleteCurrentForm();
-      this.$router.push({
-        name: 'UserForms',
-      });
-    },
-    enableSettingsEdit() {
-      if (this.settingsPanel === undefined) this.settingsPanel = 0;
-      this.formSettingsDisabled = false;
-    },
-    async updateSettings() {
-      try {
-        if (this.$refs.settingsForm.validate()) {
-          await this.updateForm();
-          this.formSettingsDisabled = true;
-          this.addNotification({
-            type: NotificationTypes.SUCCESS,
-            message: 'Your form settings have been updated successfully.',
-          });
-          this.fetchForm(this.f);
-        }
-      } catch (error) {
-        this.addNotification({
-          message:
-            'An error occurred while attempting to update the settings for this form.',
-          consoleError: `Error updating settings for ${this.form.id}: ${error}`,
-        });
-      }
-    },
   },
   mounted() {
     // Get the form for this management page
     this.fetchForm(this.f);
+    this.fetchDrafts(this.f);
     // Get the permissions for this form
     this.getFormPermissionsForUser(this.f);
```

Comment:
With so many things happening while this page is mounting, it may be worth considering putting in either a skeleton loader or equivalent while things are still loading.

Reply:
Need a structure for all page loading have some ideas for discussion later...